core-tests/shared/src/test/scala/zio/ChunkSpec.scala

```diff
@@ -60,15 +60,15 @@ object ChunkSpec extends ZIOBaseSpec {
     testM("apply") {
       check(chunkWithIndex(Gen.unit)) {
         case (chunk, i) =>
-          assert(chunk.apply(i))(equalTo(chunk.toSeq.apply(i)))
+          assert(chunk.apply(i))(equalTo(chunk.toList.apply(i)))
```

Comment:
We can omit the whole `.toList.`and use the `apply` directly on Chunk.

Reply:
Ah never mind, it's testing that it's the same as List implementation.

Reply:
I think the goal in this test was to make sure the apply method itself is accurate, and so we are comparing the `apply` implementation in `Chunk` with the known correct value from accessing the same index in the list. I suppose maybe it would be better to check against `Vector` here.